docker-compose.yaml

```diff
@@ -261,6 +261,23 @@ services:
         condition: service_started
         required: true
 
+  atoma-node-sglang:
```

Comment:
Why ?

Reply:
I suppose this is still relevant for now but eventually will be removed when `vllm` is removed, correct @jorgeantonio21 ?